build-info-extractor/src/main/java/org/jfrog/build/extractor/xrayScanViolationsTable/ViolationTableElement.java

```diff
@@ -0,0 +1,51 @@
+package org.jfrog.build.extractor.xrayScanViolationsTable;
+
+import org.apache.commons.lang.StringUtils;
+
+import java.util.Objects;
+
+/***
+ * Struct to hold a vulnerability table element.
+ */
+public class ViolationTableElement {
```

Comment:
What do you same we rename *ViolationTableElement* to *BuildScanTableElement*?
We can then also rename the package from *xrayScanViolationsTable* to *buildScanTable* and the *XrayViolationsTableHelper* class to *BuildScanTableHelper*.